spinnman/exceptions.py

```diff
@@ -229,23 +242,25 @@ def __init__(self, trace_back, connection):
     def finalise(self):
         self.chip_core += "]"
 
-    def add_coord(self, sdp_header):
-        self.chip_core += "{}[{}:{}:{}]".format(
+    def add_coord(self, sdp_header, phys_p):
+        self.chip_core += "{}[{}:{}:{}({})]".format(
             self._separator,
             sdp_header.destination_chip_x,
             sdp_header.destination_chip_y,
-            sdp_header.destination_cpu)
+            sdp_header.destination_cpu, phys_p)
         self._separator = ","
 
     @staticmethod
-    def group_exceptions(error_requests, exceptions, tracebacks, connections):
+    def group_exceptions(error_requests, exceptions, tracebacks, connections,
+                         transceiver):
         """ Groups exceptions into a form usable by an exception.
 
         :param list(SCPRequest) error_requests: the error requests
         :param list(Exception) exceptions: the exceptions
         :param list tracebacks: the tracebacks
         :param list connections:
             the connections the errors were associated with
+        :param Machine machine: the machine used
```

Comment:
This parameter should be the transceiver, not the machine?

Reply:
Fixed